recipe/meta.yaml

```diff
@@ -1,43 +1,55 @@
 {% set name = "weave" %}
-{% set version = "0.17.0" %}
-{% set sha256 = "2703f3ae6d23ad47b5f09b6bcb7affd0fb587120a0c973e7be40ef24de709998" %}
+{% set version = "0.51.31" %}
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - python_requires.patch
+  sha256: 66df71c5865a631986c346d24c7f9b1d22126bd266ad59d1610f26aa7487b025
 
 build:
-  number: 4
-  skip: True  # [py3k or not x86_64]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 0
+  skip: true  # [py<39]
+  # we don't have 'wandb' for s390x
+  skip: true  # [s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
 
 requirements:
   host:
     - python
-    - numpy
+    - hatchling
     - pip
   run:
     - python
-    - numpy
-    - {{ compiler("cxx") }}  # [not win]
+    - pydantic >=2.0.0
+    - wandb >=0.17.1
+    - packaging >=21.0
+    - tenacity >=8.3.0,!=8.4.0
+    - emoji >=2.12.1
+    - uuid-utils >=0.9.0
+    - numpy >1.21.0
+    - rich
+    - gql
```

Comment:
Expand it to 
```
install_aiohttp_requires = [
    "aiohttp>=3.8.0,<4;python_version<='3.11'",
    "aiohttp>=3.9.0b0,<4;python_version>'3.11'",
]

install_requests_requires = [
    "requests>=2.26,<3",
    "requests_toolbelt>=1.0.0,<2",
]
```
see https://github.com/graphql-python/gql/blob/v3.5.0/setup.py#L41-L49C2